donation-tracker/src/components/campaignCard.tsx

```diff
@@ -11,12 +11,22 @@ import { DonationItem } from '../types/donationItem';
 import DonationRow from './donationRow';
 import DonationPill from './donationPill';
 import { useTranslation } from 'react-i18next';
+import PageConfiguration from '../config.template';
```

Comment:
Was this on purpose? I think it's an accident and should be the config itself, or?
The build copies the template to a real one, so it will not break because of that)
```suggestion
import PageConfiguration from '../config';
```